src/trio/_tests/test_ssl.py

```diff
@@ -411,21 +417,24 @@ def ssl_wrap_pair(
         client_transport,
         client_ctx,
         server_hostname="trio-test-1.example.org",
-        **client_kwargs,
+        **client_kwargs,  # type: ignore[arg-type]
     )
```

Comment:
swapping an `Any` for a `# type: ignore[arg-type]` is borderline making it worse imo, especially in such a short function and it being a test function that nobody will call.

At least add a comment explaining why it's necessary to ignore

Reply:
Added comment in c4fccd0